app/services/TwoStepVerification.scala

```diff
@@ -38,8 +39,10 @@ trait TwoStepVerification {
 
   def twoStepVerificationThrottle: TwoStepVerificationThrottle
 
-  private val conditionsByDestination = Map(
-    BusinessTaxAccount -> List(not(HasStrongCredentials), GGEnrolmentsAvailable, HasOnlyOneEnrolment, HasSelfAssessmentEnrolments, not(HasRegisteredFor2SV))
+  case class Biz2SVRule(name: String, conditions: List[Condition])
+
+  private val biz2svRulesByDestination = Map(
```

Comment:
If the only rules this component is handling are _business_ 2sv, then this component should be probably renamed to something more specific. In this case we wouldn't need this map, as this rule are applied only when destination is bta. Any thought?